docs/license.rst

```diff
@@ -0,0 +1,4 @@
+License
+=======
+
+.. include:: ../LICENSE
```

Comment:
Just wondering, is this page necessary? I can't think of another open source project that puts the license in the docs.